src/clib/Make.mach.tacc-stampede2-intel

```diff
@@ -0,0 +1,98 @@
+#=======================================================================
```

Comment:
This file kind of duplicates `Make.mach.tacc-stampede-intel`, right? So I'm not sure it makes sense to add this (especially since TACC is moving to stampede3)